### DISCUSSION TITLE
Custom theme structure

### DISCUSSION THREAD

**Discussion by sjfernandez** [prasanthrangan/hyprdots#1581] [Q&A]
I'm reading the theming guide and it says you can fork one of the prebuilt themes but the structure seems different but it seems to be different than the one in the wiki guide

https://github.com/prasanthrangan/hyde-themes/tree/Gruvbox-Retro/Configs/.config/hyde/themes/Gruvbox%20Retro

https://github.com/prasanthrangan/hyprdots/wiki/Theming

Which one is correct? and if the first is wrong where do I find the Gruvbox Retro theme so I can make my own, also what are the files in the sources folder? do they need to be edited also?

**Comment by kRHYME7** [comment]
wiki is  outdated. the themes are inside ` ~/.config/hyde/themes `  
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/420ef51d-846f-457c-ba99-e61bdeed5056)

you can follow the structure for each theme on that folder. 

And see this repo https://github.com/kRHYME7/hyde-gallery for the list of working themes shared by the other users. You can share yours if you're done. 


Example if you want to follow the file structure of this theme https://github.com/prasanthrangan/hyde-themes/tree/Catppuccin-Mocha

**Comment by sjfernandez** [comment]
Thanks, i did think that was the case, what about the source files?

**Reply by kRHYME7** [reply]
If you're pertaining to the archived files you can search online for the sources. Try it in your system if it works/looks good. Then compress and add it to Source.  supported compression formats are tar.gz and tar.xz. 


Basically it is the cursor, gtk icons and the gtk theme. gtk theme.  only gtk-theme is mandatory.
